tests/test_photoCalib.py

```diff
@@ -46,6 +46,8 @@ def computeMagnitudeErr(instFluxErr, instFlux, calibrationErr, calibration, flux
 class PhotoCalibTestCase(lsst.utils.tests.TestCase):
 
     def setUp(self):
+        np.random.seed(100)
```

Comment:
Not global: `setUp` runs before each test, so they all get the same random seed, doesn't matter whether run in serial or parallel.

Reply:
I read @kfindeisen's point as being that `np.random.seed` isn't threadsafe: if two tests are running in the same process, they share a global RNG and can interfere with each other.

I'm not sure that pytest will ever actually execute multiple tests in the same process, though. Mind you, I'm not sure that it won't, either...

Reply:
pytest doesn't use threading. It runs tests sequentially in a single process or else it forks off subprocesses and tests are run sequentially in those.

Reply:
We don't run pytest in threads, we run it in processes, which don't see each other. There wouldn't be a point to run separate pytest tests in threads because of the GIL. 

This is the way we write all of our np-seeded tests.

Reply:
Thread-safety is irrelevant; shared (mutable) state is what I was worried about. Glad to hear nothing is shared.

Reply:
Moreover (re-thread safety): if you're objecting to this, you should object to the entire `unittest` model, since it assumes that each test is run bracketed by its own `setUp` and `tearDown`, and does not interact with any other test. That's inherently incompatible with running in shared-state threads.

Reply:
I don't understand your comment. It's perfectly possible to write thread-safe tests.